src/index.ts

```diff
@@ -1,14 +1,28 @@
 import express from 'express';
 import admin from 'firebase-admin';
+import { firebase as client } from '@firebase/app';
+import '@firebase/auth';
```

Comment:
hey aneesh, do you remember why this is being imported at the index.ts level?

Reply:
@AneeshJalan

Reply:
I felt that this may be used in some other services/controllers as well. No specific reason though for importing at the index.ts level.